services/postgres/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface SecretDict {
```

Comment:
Maybe we could be more explicit commenting or describing types here to explain whats expected. What I see from the other script docs
  
   {
        'engine': <required: must be set to 'postgres'>,
        'host': <required: instance host name>,
        'username': <required: username>,
        'password': <required: password>,
        'dbname': <optional: database name, default to 'postgres'>,
        'port': <optional: if not specified, default port 5432 will be used>
    }